src/main/java/software/coley/android/xml/XmlDecoder.java

```diff
@@ -182,6 +183,31 @@ public String getReconstructedXml() {
 		return builder.toString();
 	}
 
+	/**
+	 * Attempts to get the name of the attribute, first from the {@link StringPoolChunk} otherwise fallback
+	 * to the {@link XmlResourceMapChunk}
+	 *
+	 * @param attribute Current XML attribute.
+	 * @return The name of the attribute, empty if resourceMap is null or the raw resource id
+	 */
+	@Nonnull
+	private String getAttributeName(@Nonnull XmlAttribute attribute) {
+		String name = attribute.name();
+		if (!(name == null || name.isEmpty()))
+			return name;
+
+		if (resourceMap == null)
+			return "";
+
+		BinaryResourceIdentifier resourceId = resourceMap.getResourceId(attribute.nameIndex());
+		name = resourceProvider.getResName(resourceId.id());
+		// See res-map.txt
```

Comment:
The `res-map` is an internal implementation provided only by our test module. Recaf will have the same implementation, but a comment here should not refer to it.

Res-map is a flat view of the Android SDK's ARSC bundle's id to name mappings for attributes, created from JADX's tooling.

Reply:
We can use 

https://android.googlesource.com/platform/frameworks/base/+/57af565fb79c006522261fc211b078d2a5efd331/core/res/res/values/public.xml